src/sql/migrations/20230907104002_cell_metadata_tables.js

```diff
@@ -0,0 +1,51 @@
+const defineDeleteCellMetadataFileIfOrphanFunc = `
```

Comment:
it looks good, but I think there might be one part missing, this is the trigger to delete a row if it is orphan but we still need a trigger to delete from s3 when a row gets deleted.

For examples on that part you can look into the migrations
`add_delete_sample_file_trigger` 
`20220804092705_plot_experiment_delete_trigger.js`
and also important make sure to look into what was changed on `20230223175139_skip_plot_s3_delete_if_path_is_null.js`
 
As far as I know those are the migrations that reference the s3 delete lambda calls (there might be others I don't know about).